c7n/resources/vpc.py

```diff
@@ -134,7 +134,7 @@ def process(self, resources, event=None):
                 for fl in flogs:
                     dest_type_match = (destination_type is None) or op(
                         fl['LogDestinationType'], destination_type)
-                    dest_match = (destination is None) or op(
+                    dest_match = (destination is None) or (not hasattr(fl, 'LogDestination')) or op(
```

Comment:
hasattr is intended for objects, for dictionaries, a key check is typically done via `if key in dict` or dict.get(key)